CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 
 ## NEXT VERSION
 
-## 20230313 (latest)
+## 20230323 (latest)
+* **GCC version: 9.3.0**
+* update host utils
+* phoenix-rtos-project commit: `9b28e9b`
```

Comment:
no, the `build` image was comiled against `e8b8ba5` and this is the version of toolchain/libphoenix, etc.